bootstrap.sh

```diff
@@ -183,17 +183,11 @@ require "socket"
 server = TCPServer.new("127.0.0.1", 15782)
 socket = server.accept
 
-PTY.open do |io, file|
-  pid = Process.spawn({"TERM" => "xterm"}, "/bin/bash", "--login", "/Users/travis/build.sh", [:out, :err] => file)
-  pipe_thread = Thread.new do
-    loop do
-      socket.print(io.read(1))
-    end
-  end
+PTY.spawn("/usr/bin/env", "TERM=xterm", "/bin/bash", "--login", "/Users/travis/build.sh") do |stdout, stdin, pid|
+  stdin.close
```

Comment:
I just tried this script on my Mac, and even with this line the script still stalled when trying to read something from stdin, so it appears it doesn't have any effect at all.